client/my-sites/comments/comment/index.jsx

```diff
@@ -47,19 +48,29 @@ export class Comment extends Component {
 		this.state = {
 			isEditMode: props.isEditMode,
 			isReplyVisible: false,
+			offsetTop: 0,
 		};
 	}
 
 	componentWillReceiveProps( nextProps ) {
 		const { isBulkMode: wasBulkMode } = this.props;
 		const { isBulkMode } = nextProps;
 
-		this.setState( ( { isEditMode, isReplyVisible } ) => ( {
+		const newOffsetTop = this.getCommentOffsetTop();
+
+		this.setState( ( { isEditMode, isReplyVisible, offsetTop } ) => ( {
```

Comment:
We can avoid setting state if props don't change, otherwise this will fire for any other prop. For example
```
if( nextProps.isBulkMode !== this.props.isBulkMode || nextProps.commentId !== this.props.commentId ) {
    const newOffsetTop = this.getCommentOffsetTop();
    this.setState({ 
//.... 
} );
}
```